SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -30,6 +32,11 @@ Worker_ComponentUpdate InterestFactory::CreateInterestUpdate()
 
 Interest InterestFactory::CreateInterest()
 {
+	if (!GetDefault<USpatialGDKSettings>()->bUsingQBI)
+	{
+		return Interest();
```

Comment:
Nit, I think we're trying to do `{}` initialization but I always forget.

Reply:
Was about to say the same. Prefer `{}` except when there is a constructor that takes an initializer list.